tmessage/auth.py

```diff
@@ -56,3 +56,21 @@ def register(user_name, displayed_name, password, password_confirm):
         return jwt.decode(data['token'], verify=False)
     error = response.json()
     raise Exception(f'Error Registering User: {error["message"]}')
+
+
+def change_display_name(user_name, displayed_name):
+    """ Change display name of a user """
+    endpoint_url = f'{API_USER_URL}/changeDisplayName'
+    headers = {
+        "Content-Type": "application/json"
+    }
```

Comment:
Hi @Nephyx,
To adapt the code to the protected route on API (I suggested this at [Haider8/tmessage-api#3](https://github.com/Haider8/tmessage-api/pull/3)), when a request to change the displayed name is made, the request's header need the `Authorization` argument with a value of `Bearer [access_token]`, too.
Whenever the user receives a token, it should be saved to a variable so that it can be sent later on in the request's header.